packages/react-native-renderer/src/ReactNativeBridgeEventPlugin.js

```diff
@@ -22,11 +22,10 @@ import {ReactNativeViewConfigRegistry} from 'react-native/Libraries/ReactPrivate
 const {
   customBubblingEventTypes,
   customDirectEventTypes,
-  eventTypes,
 } = ReactNativeViewConfigRegistry;
 
 const ReactNativeBridgeEventPlugin = {
-  eventTypes: eventTypes,
+  eventTypes: {},
```

Comment:
Note: even though the docstring says this is optional, the [flowtype](https://github.com/facebook/react/blob/a210b5b44016c086f83008cd03146d5acdacd12c/packages/events/PluginModuleType.js#L24) disagress